conf/base.py

```diff
@@ -163,6 +164,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'simple_history.middleware.HistoryRequestMiddleware',
```

Comment:
Since the locations objects are created before the shipment object, in order to assign the `author` of a change to a location, we either need to add this middleware: `simple_history.middleware.HistoryRequestMiddleware` to retrieve the user from the `request` or add the column `updated_by` to Location table and set its value manually in shipment’s serializer.